styles.scss

```diff
@@ -247,3 +258,66 @@ $color-link-bold-dark: lighten($color-link-bold, 20%);
     font-style: italic;
   }
 }
+
+
+// Stickybox ad
+// -------------------------------------------------------------------------
+// The stickybox is an ad that is shown in a floating box in the lower right
+// on very wide screens or a standard image ad
+// (text-only not supported) on screens less wide.
+// https://ethical-ad-client.readthedocs.io/en/latest/#stickybox
+[data-ea-style="stickybox"].loaded {
+  .ea-type-image {
+    z-index: 1000;
```

Comment:
There was some conversation about this z-index. It makes the ad cover some contents on the screen in the RTD app
![Screenshot_2021-11-29_16-55-04](https://user-images.githubusercontent.com/25510/144328039-1c9c6230-f97e-44ca-adc5-068d9adeb074.png).

I'm curious if we should use a less aggressive z-index?

Reply:
I'm not sure what the solution is here. Since the sphinx-search is a plugin and could define any z-index, what do you even think the z-index should be? We don't want random elements hiding the ad.

Reply:
Yea, it's always going to be a trade-off. I wish there was some standard for z-index's like for log levels in Python. I don't really know what a "correct" z-index would be..

I think "live" elements probably should hide the ad (it's part of the page content in my thinking), but yea there's no perfect solution.

Reply:
I just checked and this doesn't seem to be controlled by the zindex (or I'm missing something). The zindex on the search modal is larger than the ad.

Reply:
It is controlled by the z-index and a different part of the search modal sets a z-index of 500. We are tweaking this to be below that: https://github.com/readthedocs/ethical-ad-client/pull/98